src/character.cpp

```diff
@@ -1280,9 +1281,13 @@ int Character::swim_speed() const
             ret -= ( 15 * str_cur ) / 2;
         }
     }
-    /** @EFFECT_STR increases swim speed bonus from WEBBED */
+    /** @EFFECT_STR increases swim speed bonus from WEBBED and WEBBED_FEET */
+    float webbing_factor = 60 + str_cur * 5;
     if( has_trait( trait_WEBBED ) ) {
-        ret -= hand_bonus_mult * ( 60 + str_cur * 5 );
+        ret -= hand_bonus_mult * webbing_factor * 0.7f;
+    }
+    if( has_trait( trait_WEBBED_FEET ) ) {
+        ret -= ( 1 - footwear_factor() ) * webbing_factor * 0.3f;
```

Comment:
```suggestion
        ret -= hand_bonus_mult * webbing_factor * 0.5f;
    }
    if( has_trait( trait_WEBBED_FEET ) ) {
        ret -= ( 1 - footwear_factor() ) * webbing_factor * 0.5f;
```
Should be 50:50 aka minimum weight for the feet should be 50%